src/components/file/DeleteFile.tsx

```diff
@@ -58,11 +61,17 @@ const DeleteFile = ({ file, concernedFiles }: DeleteFileProps): JSX.Element => {
 					w="100%"
 					p="0px"
 					mx="4px"
-					onClick={() => onOpen()}
+					onClick={() => {
```

Comment:
would be better to put this in a dedicated function outside the tags :smile:

Reply:
Done